integration-tests/src/test/java/oracle/weblogic/kubernetes/ItLivenessProbeCustomization.java

```diff
@@ -231,6 +243,313 @@ void testCustomLivenessProbeNotTrigged() {
+    assertTrue(successThreshold.intValue() == 1, "The original livenessProbe successThreshold is not 1");
+
+    // patch the domain with custom livenessProbe successThreshold
+    String patchStr =
+        "[{\"op\": \"replace\", \"path\": \"/spec/serverPod/livenessProbe/successThreshold\", \"value\": 2}]";
+    logger.info("Updating domain configuration using patch string: {0}", patchStr);
+    assertTrue(patchDomainCustomResource(domainUid, domainNamespace, new V1Patch(patchStr), PATCH_FORMAT_JSON_PATCH),
+        String.format("failed to patch domain %s in namespace %s", domainUid, domainNamespace));
+
+    // check the operator log contains expected error msg
+    String expectedErrorMsg = "Invalid value '2' for the liveness probe success threshold under "
+        + "'spec.adminServer.serverPod.livenessProbe.successThreshold'. "
+        + "The liveness probe successThreshold value must be 1.";
+    String operatorPodName = assertDoesNotThrow(() -> getOperatorPodName(OPERATOR_RELEASE_NAME, opNamespace));
+    checkPodLogContainsString(opNamespace, operatorPodName, expectedErrorMsg);
+
+    // check the domain status conditions failed and condition message contains the expected error msg
+    verifyDomainStatusCondition(domainUid, domainNamespace, "Failed", expectedErrorMsg);
+
+    // set the domain back to original state
+    // patch the domain with the original livenessProbe successThreshold
+    patchStr = "[{\"op\": \"replace\", \"path\": \"/spec/serverPod/livenessProbe/successThreshold\", \"value\": 1}]";
+    logger.info("Updating domain configuration using patch string: {0}", patchStr);
+    assertTrue(patchDomainCustomResource(domainUid, domainNamespace, new V1Patch(patchStr), PATCH_FORMAT_JSON_PATCH),
+        String.format("failed to patch domain %s in namespace %s", domainUid, domainNamespace));
+
+    domain1 = assertDoesNotThrow(() -> getDomainCustomResource(domainUid, domainNamespace),
+        String.format("getDomainCustomResource failed with ApiException when tried to get domain %s in namespace %s",
+            domainUid, domainNamespace));
+    assertNotNull(domain1, "Got null domain resource");
+    assertNotNull(domain1.getSpec(), "domain1.getSpec() is null");
+    assertNotNull(domain1.getSpec().getServerPod(), "domain1.getSpec().getServerPod() is null");
+    assertNotNull(domain1.getSpec().getServerPod().getLivenessProbe(),
+        "domain1.getSpec().getServerPod().getLivenessProbe() is null");
+
+    // get the successThreshold of livenessProbe
+    successThreshold = domain1.getSpec().getServerPod().getLivenessProbe().getSuccessThreshold();
+    logger.info("The livenessProbe successThreshold is: {0}", successThreshold);
+    assertTrue(successThreshold.intValue() == 1, "The livenessProbe successThreshold is not 1");
+  }
+
+  /**
+   * Patch the domain with custom readinessProbe failureThreshold and successThreshold value in serverPod.
```

Comment:
Does this test not cover earlier two test methods testCustomLivenessProbeSuccessThreshold() and testCustomLivenessProbeFailureThreshold()  ?

Reply:
Can we check runtime behavior of these two new attributes ?

Reply:
This is for readinessProbe. The other two tests are for livenessProbe. 
I will add the runtime check for this two attribute.

Reply:
Can u not combine testCustomLivenessProbeSuccessThreshold() and testCustomLivenessProbeFailureThreshold() into a single test method testCustomLivenessProbeFailureThresholdSuccessThreshold()

Reply:
Maybe we can